### PR TITLE
Intergrated swish api into backend

### PR DESCRIPTION
**Pull request by Deltaphish**
This branch creates a swish payment request when a order is made, and allows the frontend to query the backend if a order has been paid or cancelled.

Since we don't (want/have) a company swish id, this implementation uses the test swish api, simulating payment.

Swish requires that all communication is securely encrypted by tls, which means that for payments to be registered the wordpress server must use tls.

The ceritfications added in this merge are publicaly available so having them in the git repo for now should be fine.

There is an error when calculating total cost,(#17) but the solution to that is out of scope for this pr.

### CODE REVIEW

File: pizzakit-plugin/includes/pizzakit.php [hunk 1]
```diff
@@ -1,8 +1,20 @@
 <?php
 
 class Pizzakit {
+	public static $resp = "None";
```
Comment (89netraM) [inline pizzakit-plugin/includes/pizzakit.php:4]: Why is this a static class variable?

Comment (Deltaphish) [reply pizzakit-plugin/includes/pizzakit.php]: That is a failed attempt to debug the rest api, and should definitly be removed

File: pizzakit-plugin/includes/pizzakit.php [hunk 2]
```diff
@@ -37,8 +53,172 @@ private static function handle_post() {
...
+		}
+		return(-1);
+	}
+
+	private static function create_swish_payment($order_id,$cost){
+		$random_uuid = str_replace("-","",wp_generate_uuid4());
+		$endpoint = "/v2/paymentrequests/".$random_uuid;
+		$method = CURLOPT_PUT;
+		$data = array(
+			"payeePaymentReference" => "0123456789",
```
Comment (89netraM) [inline pizzakit-plugin/includes/pizzakit.php:138]: Order ID

File: pizzakit-plugin/includes/pizzakit.php [hunk 3]
```diff
@@ -37,8 +53,172 @@ private static function handle_post() {
...
+			"callbackUrl" => get_home_url() . "/index.php/wp-json/pizzakit/callback/" . $order_id,
+			"payerAlias" => "4671234768",
```
Comment (89netraM) [inline pizzakit-plugin/includes/pizzakit.php:140]: Number from form

**Pull request merged**